src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGetServices.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.ServiceHub.Framework;
+using StreamJsonRpc;
+
+namespace NuGet.VisualStudio.Contracts
+{
+    /// <summary>Constants and service descriptors to use NuGet.VisualStudio.Contracts extensibility.</summary>
+    public static class NuGetServices
+    {
+        /// <summary>Service name for the NuGetProjectService service.</summary>
+        public const string NuGetProjectServiceName = "Microsoft.VisualStudio.NuGet.NuGetProjectService";
```

Comment:
fwiw, I don't know if there's a best practices suggested for this, but for the internal service I just used the name. 

https://github.com/NuGet/NuGet.Client/blob/070c0166678caf816d4bcf7c9a4d2b168287e1d0/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs#L11

This makes it a bit more explicit though, so I'm ok with it.

Reply:
Looking at VS's `VS.RPC.Contracts` source, all of the services defined there use the namespace style `Microsoft.VisualStudio.Component` style naming. See `VisualStudioServices*.cs` in that repo.

Reply:
Works for me :)